### PR TITLE
made coloring tool less op

### PR DESCRIPTION
**Pull request by BigXplosion**

### CODE REVIEW

File: src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java [hunk 1]
```diff
@@ -33,9 +36,11 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class ItemPipeColoringTool extends Item {
+public class ItemPipeColoringTool extends Item implements IEnergyContainerItem {
 
 	public static final String[] names = new String[]{"white", "orange", "magenta", "light_blue", "yellow", "lime", "pink", "gray", "light_gray", "cyan", "purple", "blue", "brown", "green", "red", "black"};
+	public static final int MAX_ENERGY = 100000;
+	public static final int COLOR_ENERGY = 10; //can be tweaked if needed
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: maybe make these a config option?

Comment (iTitus) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: Thats what I thought too

File: src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java [hunk 2]
```diff
@@ -33,9 +36,11 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class ItemPipeColoringTool extends Item {
+public class ItemPipeColoringTool extends Item implements IEnergyContainerItem {
```
Comment (iTitus) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: Cant you just extend ItemEnergyContainer or itemBase? Or create a new ItemPowered class?

Comment (iTitus) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: Thats because there is so much doubled code...

Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: yeah, we need one base class for all powered tools, else we're gona have alot of duplicate code

Comment (iTitus) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: duplicate was the word I was looking for :P

Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: is any of you guys writing this code or should i do it?

Comment (iTitus) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: Its simply copying from ItemEnergyContainer... I am working on this configurable updates thing and @BigXplosion has this PR open. I would say @BigXplosion should do this.

Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java]: doesn't matter to much who does it, as long as someone does it and it get's done right

**Comment by BigXplosion** [comment]
noo, merge conflicts now :/

**Pull request closed without merging**